joomla/joomla-cms
# [com_fields] Remove unneeded fields from com_fields

Pull Request for Issue #12733.

### Summary of Changes
* Removes `image` and `image alternate text` fields from the field definition. Those aren't used anywhere and would only confuse users.
* Removes `options`, `attributes`, `created_by_alias`, `publish_up` and `publish_down` columns from the #__fields table. They are not used at all and make no sense in the context of a field.

### Testing Instructions
* Test that creating, editing and general use of fields work as before. There should be no difference except that the two "image" and "image alternate text" fields are no longer part of the edit form.
* To test the database changes, you need to either install a fresh installation or update from a 3.6.5 installation to this branch. So this may be a bit tricky. You can however also manually delete the columns and test or just create a few fields and verify there is no data stored into those columns.

### Documentation Changes Required
None
